cheddar/cheddar-integration-aws/build.gradle

```diff
@@ -2,21 +2,22 @@ apply from: '../../test.gradle'
 apply from: '../../logging-api.gradle'
 
 def springVersion = '4.1.1.RELEASE'
+def awsSdkVersion = '1.11.6'
```

Comment:
You can check the actual dependencies used in any Gradle project by using the `dependencies` task e.g
`./gradlew :iam:dependencies --configuration runtime`